staging/src/k8s.io/apimachinery/pkg/runtime/serializer/codec_factory.go

```diff
@@ -96,6 +108,8 @@ func newSerializersForScheme(scheme *runtime.Scheme, mf json.MetaFactory) []seri
 // CodecFactory provides methods for retrieving codecs and serializers for specific
 // versions and content types.
 type CodecFactory struct {
+	options CodecFactoryOptions
```

Comment:
I'm not seeing where this is used... am I missing it?

Reply:
Good catch! :pray: 
This was used in an older implementation.
Removed now :heavy_check_mark: